### PR TITLE
KB - liocheck error message

### PR DESCRIPTION
**Pull request by hubvu**

### CODE REVIEW

File: solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+```bash
+# Truncated output.
+category=lio level=WARN   msg="Runtime error checking stage 'Start TCMU and create device': /sys/module/target_core_user is missing, is kernel configfs present and target_core_user loaded?"
+category=lio level=FATAL  msg="liocheck: FAIL (platform not supported, see previous error messages)"
+```
+
+## Root Cause
+
+The root cause of this issue is due to one or more of the required [Linux-IO (LIO) related kernel modules](https://en.wikipedia.org/wiki/LIO_%28SCSI_target%29) have not been successfully loaded onto the worker nodes were the Ondat daemonset will run.
```
Comment (cannischan) [inline solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md:18]: were -> where

File: solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Resolution
+
+1. Ensure that the following kernel modules are are loaded onto the worker nodes where an Ondat daemonset pod will run.
```
Comment (cannischan) [inline solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md:22]: remove extra 'are'

**Review by cannischan — COMMENTED** [review]
small changes needed

**Comment by hubvu** [comment]
> small changes needed

fixed!

File: solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ ```bash
+ # Ensure that "kmod" is installed.
+ sudo apt install kmod               # Debian based distributions.
+ sudo dnf install kmod               # Red Hat based distributions.
+
+ # Use "modprobe" to load the kernel modules below on the worker nodes were Ondat will run.
+ modprobe --all target_core_mod tcm_loop configfs target_core_user uio
+ ```
+
+    > 💡 For more information on the required kernel modules for Ondat, review the [Ondat Prerequisites](https://github.com/ondat/documentation/blob/kb-ondat-init-error-after-deployment/docs/prerequisites/systemconfiguration) page.
```
Comment (cannischan) [inline solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md:40]: It is breaking the linter:

```
/github/workspace/solution-troubleshooting-liocheck-FAIL-platform-not-supported-see-previous-error-messages-after-deploying-ondat.md:40 MD046/code-block-style Code block style [Expected: fenced; Actual: indented]
```

**Review by cannischan — APPROVED** [review]

**Pull request merged**